sherpa/conftest.py

```diff
@@ -650,5 +650,25 @@ def cleanup_ds9_backend():
 
 @pytest.fixture(autouse=True)
 def add_sherpa_test_data_dir(doctest_namespace):
-    doctest_namespace["data_dir"] = 'sherpa-test-data/sherpatest/'
+    '''Define `data_dir` for doctests
+
+    We make this an autouse=True fixture, because that means that
+    the variable `data_dir` is available in every file that we may
+    want to test without any special markup to that file. There is a
+    risk that this is too magical and could confuse developers
+    in the future, but it seems more important that this keeps any
+    extra markup out of those files and keep them clean.
+    '''
     doctest_namespace["data_3c273"] = 'sherpa/astro/datastack/tests/data/'
+
+    # We could error out here, but we only want the tests that
+    # use it to fail.
+    #
+    path = get_datadir()
+    if path is None:
+        pytest.skip('this doctest does not work on Windows')
```

Comment:
Err. I think this message is a tad mis-leading ;-)

Reply:
But it's true!
(and taken from here: https://docs.pytest.org/en/7.1.x/how-to/doctest.html#skipping-tests)